package.json

```diff
@@ -17,6 +17,7 @@
   },
   "scripts": {
     "lint": "lerna run lint",
-    "postinstall": "lerna bootstrap"
+    "postinstall": "lerna bootstrap",
+    "build": "rollup --config"
```

Comment:
does root repo has/need a build?

Reply:
No, shouldn't be there. Will remove that line